internal/grpc/services/appprovider/appprovider.go

```diff
@@ -38,14 +46,20 @@ func init() {
 
 type service struct {
 	provider app.Provider
+	conf     *config
 }
 
 type config struct {
-	Driver string                 `mapstructure:"driver"`
-	Demo   map[string]interface{} `mapstructure:"demo"`
+	Driver    string                 `mapstructure:"driver"`
+	Demo      map[string]interface{} `mapstructure:"demo"`
+	IopSecret string                 `mapstructure:"iopsecret" docs:"The iopsecret used to connect to the wopiserver."`
```

Comment:
If you're passing only one value, the docs tag expects it to be the default https://github.com/cs3org/cato#extraction. If you want to add the description, it should be the second argument, or should be added as a comment

Reply:
Also, please run make after adding the tags, otherwise documentation won't be generated.